Move constants defined in "coordinates" sub-package to "sun" (#4013)
sunpy/sunpy: changelog/4013.feature.rst

Added extra constants to `sunpy.sun.constants`:

> - Longitude of the prime meridian (epoch J2000.0) : `sunpy.sun.constants.W_0`

Reviewer: ```suggestion
- Longitude of the prime meridian (epoch J2000.0) : ``sunpy.sun.constants.get('W_0')``
```

- Sidereal rotation rate : `sunpy.sun.constants.sidereal_rotation_rate`
- First Carrington rotation (JD TT) : `sunpy.sun.constants.first_carrington_rotation`
- Mean synodic period : `sunpy.sun.constants.mean_synodic_period`

> - Right ascension (RA) of the north pole (epoch J2000.0) : `sunpy.sun.constants.alpha_0`
> - Declination of the north pole (epoch J2000.0) : `sunpy.sun.constants.delta_0`

Reviewer: ```suggestion
- Right ascension (RA) of the north pole (epoch J2000.0) : ``sunpy.sun.constants.get('alpha_0')``
- Declination of the north pole (epoch J2000.0) : ``sunpy.sun.constants.get('delta_0')``
```

Added extra constants to `sunpy.sun.constants`:

> - True longitude of meridian : `sunpy.sun.constants.meridian_lon`
> - Sidereal rotation rate : `sunpy.sun.constants.sidereal_rotation_rate`
> - First Carrington rotation : `sunpy.sun.constants.first_crot_jd`
> - First Carrington rotation (JD TT) : `sunpy.sun.constants.first_crot_jd_tt`
> - Mean synodic period : `sunpy.sun.constants.crot_period`
> - Right Ascension (RA) of the north pole at epoch J2000.0 : `sunpy.sun.constants.sun_north_pole_lon`
> - Declination of the North Pole at epoch J2000.0 : `sunpy.sun.constants.sun_north_pole_lat`

Reviewer: ```suggestion
- Longitude of the prime meridian (epoch J2000.0) : `sunpy.sun.constants.W_0`
- Sidereal rotation rate : `sunpy.sun.constants.sidereal_rotation_rate`
- First Carrington rotation (JD TT) : `sunpy.sun.constants.first_carrington_rotation`
- Mean synodic period : `sunpy.sun.constants.mean_synodic_period`
- Right ascension (RA) of the north pole (epoch J2000.0) : `sunpy.sun.constants.alpha_0`
- Declination of the north pole (epoch J2000.0) : `sunpy.sun.constants.delta_0`
```
